src/pages/instantMessenger/messages/messages.tmpl.js

```diff
@@ -0,0 +1,17 @@
+// language=hbs
+
+export default `
+<div class="messages-layout">
+    {{{header}}}
+    <section>
+        <ul class='messages'>
+            {{#each messages}}
+                <li class='message {{#if this.author}}message_outcoming{{else}}message_incoming{{/if}}'>
```

Comment:
Можно сократить, а так же булевы переменные лучше именовать с `is`, например isAuthor, isOwn. Это улучшает читаемость кода
```suggestion
                <li class='message message_{{#if this.author}}outcoming{{else}}incoming{{/if}}'>
```